static/js/http.js

```diff
@@ -1,3 +1,30 @@
+const connectionStatus = {
+  isConnected: true,
+  lastSuccess: Date.now(),
```

Comment:
`lastSuccess` is never used, right? same functionality seems to be handled by `disconnectedAt` in `layout.js`